Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/NamespaceSynchronizationConfig.swift

```diff
@@ -26,8 +26,9 @@ internal struct NamespaceSynchronization: Sequence {
             self.namespace = namespace
             self.syncedDocuments = syncedDocuments
         }
+
         static func == (lhs: NamespaceSynchronization.Config, rhs: NamespaceSynchronization.Config) -> Bool {
-            return true
+            return lhs.namespace == rhs.namespace
```

Comment:
[q] when are we actually using the comparison? I think if we don't need it, we should get rid of it instead of having a shallow comparison

Reply:
We may use it down the line, later in our development process. I can remove it for now.

Reply:
yeah let's remove it until we actually need it, and we can decide what makes two configs equal then

Reply:
not needed in tests?